Chat App/View/Pub/PubChatLoginViewController.swift

```diff
@@ -0,0 +1,103 @@
+        passwordField.font = UIFont.systemFont(ofSize: 15)
+        passwordField.backgroundColor = .white
+        passwordField.isSecureTextEntry = true
+        passwordField.borderStyle = .roundedRect
+        passwordField.placeholder = Constants.passwordPlaceholderText
+        view.addSubview(passwordField)
+        
+        // Login button
+        loginButton = UIButton()
+        loginButton.translatesAutoresizingMaskIntoConstraints = false
+        loginButton.setTitle(Constants.loginButtonTitle, for: .normal)
+        loginButton.backgroundColor = .black
+        loginButton.layer.cornerRadius = 9.0
+        loginButton.addTarget(self, action: #selector(self.loginButtonTapped), for: .touchUpInside)
+        view.addSubview(loginButton)
+        
+        // Sign Up button
+        registerButton = UIButton()
+        registerButton.translatesAutoresizingMaskIntoConstraints = false
+        registerButton.setTitle(Constants.registerButtonTitle, for: .normal)
+        registerButton.backgroundColor = .black
+        registerButton.layer.cornerRadius = 9.0
+        registerButton.addTarget(self, action: #selector(self.registerButtonTapped), for: .touchUpInside)
+        view.addSubview(registerButton)
+        
+        NSLayoutConstraint.activate ([
+            bannerLabel.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: 40),
+            bannerLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            
+            userNameField.topAnchor.constraint(equalTo: bannerLabel.bottomAnchor, constant: 70),
+            userNameField.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            userNameField.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.7),
+            
+            passwordField.topAnchor.constraint(equalTo: userNameField.bottomAnchor, constant: 20),
+            passwordField.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            passwordField.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.7),
+            
+            loginButton.topAnchor.constraint(equalTo: passwordField.bottomAnchor, constant: 70),
+            loginButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            loginButton.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.7),
+            loginButton.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 0.07),
+            
+            registerButton.topAnchor.constraint(equalTo: loginButton.bottomAnchor, constant: 15),
+            registerButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            registerButton.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.7)
+        ])
+    }
+    
+    @objc func loginButtonTapped() {
+        print("PubChat Login button tapped.")
+    }
+    
+    @objc func registerButtonTapped() {
+        print("PubChat register button tapped.")
+    }
```

Comment:
These print() placeholders will be replaced with actual navigation code in the next pull request.